pages/common/k8sec.md

```diff
@@ -0,0 +1,32 @@
+# k8sec
+
+> Command line interface tool to manage Kubernetes secrets.
+> More information: <https://github.com/dtan4/k8sec>.
+
+- List all secrets:
+
+`k8sec list`
+
+- List specific secret and decode it with base64:
+
+`k8sec list {{secret_name}} --base64`
+
+- Set secret:
+
+`k8sec set {{secret_name}} {{key=value}}`
+
+- Set base64-encoded value:
+
+`k8sec set --base64 {{secret_name}} {{key=encoded_value}}`
+
+- Unset secret:
+
+`k8sec unset {{secret_name}}`
+
+- Load secrets from dotenv file:
+
+`k8sec load -f {{file_name}} {{secret_name}}`
```

Comment:
Perhaps we could alter `file_name` here to make it more consistent with other pages?
```suggestion
`k8sec load -f {{path/to/file}} {{secret_name}}`
```

...the same goes for the example below.